README.md

```diff
@@ -1,2 +1,42 @@
 # CSCI 420 (Spring 2021) MontyPython UML Editor
 
+This command line program creates a text representation of a UML diagram though user input. The program’s capabilities include adding, deleting, displaying, and modifying UML diagram elements. It also supports the ability to save and load diagrams.
+
+## Prerequisites
+1. To run the program, [Python 3.9.1](https://www.python.org/downloads/) needs to be installed.
+1. During installation, check the box labeled "Add Python 3.9 to PATH" so Python programs can be run by typing `Python` in a terminal.
```

Comment:
Lowercase `python` here at the end of the sentence since the default command name is lowercase.